pkgs/development/python-modules/amulet-leveldb/default.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  zlib,
+
+  # build-system
+  setuptools,
+  wheel,
+  cython,
+  versioneer,
+
+  pytestCheckHook,
+  nix-update-script,
+}:
+let
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "Amulet-Team";
+    repo = "Amulet-LevelDB";
+    rev = "refs/tags/${version}";
+    hash = "sha256-7VJb5TEa3GcwNEZYrnwP3yMWdpxkHeFcydCCeHiB3+w=";
+
+    postFetch = ''
+      # De-vendor zlib
+      rm -r $out/zlib
+    '';
+    fetchSubmodules = true;
+  };
+in
+buildPythonPackage {
+  pname = "amulet-leveldb";
+  inherit version src;
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail "versioneer.get_version()" "'${version}'"
+  '';
+
+  build-system = [
+    setuptools
+    wheel
+    cython
+    versioneer
+  ];
+
+  buildInputs = [ zlib.dev ];
```

```suggestion
  buildInputs = [ zlib ];
```

We shouldn't need to select the output in this context